templates/rzp-mini-checkout-btn.php

```diff
@@ -1,10 +1,12 @@
-<div>
-    <button
-      id="btn-1cc-mini-cart"
-      class="button checkout wc-forward"
-      type="button"
-      >CHECKOUT
-    </button>
+<div id="btn-1cc-mini-cart" >
+   <magic-checkout-btn page-type="cart_sm" border-radius="0px"> ></magic-checkout-btn>
 </div>
-<div id="error-message">
+<div id="rzp-spinner-backdrop"></div>
+<div id="rzp-spinner">
+  <div id="loading-indicator"></div>
+  <div id="icon">
+    <img src="<?php echo plugin_dir_url(dirname(__FILE__)) . 'public/images/rzp-spinner.svg'; ?>" alt="Loading"  id="rzp-logo" />
```

Comment:
please remove the spinner code from here as well, It should only be there in the separate file.